doc/hacking/bugs/bgo-650785/bisect.sh

```diff
@@ -1,10 +1,10 @@
 GOOD=/home/thomas/gst/git/gst-plugins-good/gst/audioparsers
-MORI=/home/thomas/dev/own/morituri
+WHIP=/home/thomas/dev/own/whipper
```

Comment:
Fixed reference to thomas’ PATH don’t look right anyway. What is this file? Just a bisect script he use(d) for some bugs?